src/main/java/controller/RequestController.java

```diff
@@ -0,0 +1,127 @@
+package controller;
+
+import db.DataBase;
+import http.HttpHeaders;
+import http.HttpQueryParams;
+import http.HttpRequest;
+import http.HttpResponse;
+import model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import utils.FileIoUtils;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public class RequestController {
+    private static final Logger logger = LoggerFactory.getLogger(RequestController.class);
+
+    public static void handleRequest(HttpRequest httpRequest, DataOutputStream dos) throws IOException {
+        String path = httpRequest.getStartLine().getPath();
+        if (isFile(path)) {
+            handleFile(httpRequest, dos);
+            return;
+        }
+
+        if (path.startsWith("/user/create")) {
+            if (httpRequest.isGet()) {
+                handleUserCreate(httpRequest.getQueryParams(), dos);
+                return;
+            }
+            if (httpRequest.isPost()) {
+                String body = (String) httpRequest.getBody();
+
+                HttpQueryParams queryParams = HttpQueryParams.parseParams(body);
+
+                handleUserCreate(queryParams, dos);
+                return;
+            }
+        }
+
+        handleRoot(dos);
+    }
+
+    private static void handleUserCreate(HttpQueryParams queryParams, DataOutputStream dos) throws IOException {
+        String userId = queryParams.get("userId");
+        String password = queryParams.get("password");
+        String name = queryParams.get("name");
+        String email = queryParams.get("email");
+        User user = new User(userId, password, name, email);
+
+        DataBase.addUser(user);
+        HttpHeaders headers = new HttpHeaders();
+        headers.put("Location", "/index.html");
+
+        HttpResponse response = new HttpResponse(HttpStatus.FOUND, headers, null);
```

Comment:
생성자 오버로딩을 통해서 body를 생성자 파라미터로 전달받지 않는 생성자를 만들어보면 어떨까요? 🙂

```suggestion
        HttpResponse response = new HttpResponse(HttpStatus.FOUND, headers);
```

Reply:
또한 추가적으로 생각해볼 수 있는 것은 redirect 같은 응답은 반환하는 Header가 고정적인 것 같아요. 🤔
이런 경우 정적 팩토리 메서드로 반환하면 재사용하기 용이하지 않을까요? 🙂

```suggestion
        HttpResponse response = HttpResponse.redirect("/index.html");
```